airgun/views/report_template.py

```diff
@@ -85,3 +85,17 @@ def is_displayed(self):
             and self.breadcrumb.locations[0] == 'Report Templates'
             and self.breadcrumb.read() == 'Create Template'
         )
+
+class ReportTemplateGenerateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    submit = Text('//input[@name="commit"]')
```

Comment:
please add two fields that form has

Reply:
Input fields are template-specific

Reply:
Then it is necessary describe them. At least most commonly used through conditional view

Reply:
It would be awful to hardcode values for some report templates. Instead, I coded EditableEntryMultiTextInput widget (PR https://github.com/SatelliteQE/airgun/pull/325) which is more general. Seems reasonable?